File: datasets/health/medical-data/modify/finish_activity.swift
import SwiftUI
struct finish_activity: View {
    
    var body: some View {
        ScrollView
        {
            ZStack
            {
                ZStack
                {
                    VStack
                    {
                        ZStack
                        {
                            Text("Thanks! Your answers have been correctly sent")
                                .padding(.leading,16)
                                .padding(.trailing,16)
                                .padding(.leading,16)
                                .padding(.top,16)
                                .padding(.bottom,16)
                        }
                        .padding(.top,16)
                        .padding(.bottom,16)
                        .padding(.leading,16)
                        .padding(.trailing,16)
                        .background(Image("border").resizable())
                        Button(action: {})
                        {
                            Text("ACCEPT")
                                .foregroundColor(.black)
                        }
                        .background(Color.white)
                        .padding(.top,80)
                    }
                }
                .padding(.top,50)
                .padding(.bottom,50)
                .padding(.leading,30)
                .padding(.trailing,30)
            }
        }}}

struct finish_activity_Previews: PreviewProvider {
    static var previews: some View {
        finish_activity()
    }
}

File: datasets_v2/health/medical-data/compose/finish_activity.kt
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.lazy.LazyColumn
import androidx.compose.foundation.lazy.items
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.paint
import androidx.compose.ui.graphics.Color
import androidx.compose.foundation.Image
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.text.style.TextAlign
import androidx.compose.ui.unit.sp
import androidx.compose.ui.unit.dp

@Composable
fun finish_activity() {
  Column(modifier = Modifier.fillMaxWidth().fillMaxHeight().verticalScroll(rememberScrollState())) {
    Box(modifier = Modifier.fillMaxWidth().fillMaxHeight()) {
      Column(modifier = Modifier.fillMaxWidth().wrapContentHeight().padding(top = 50.0.dp).padding(bottom = 50.0.dp), verticalArrangement = Arrangement.Top, horizontalAlignment = Alignment.Start) {
        Box(modifier = Modifier.paint(painterResource(id = R.drawable.border), contentScale = ContentScale.Crop).fillMaxWidth().wrapContentHeight().padding(top = 16.0.dp).padding(bottom = 16.0.dp)) {
          Text(text = "Thanks! Your answers have been correctly sent", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(top = 16.0.dp).padding(bottom = 16.0.dp).padding(start = 16.0.dp).padding(end = 16.0.dp))
        }
        Button(onClick = {}, enabled = true, modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(top = 80.0.dp)) {
          Text(text = "ACCEPT")
        }
      }
    }
  }
}


@Preview(showBackground = true)
@Composable
fun finish_activity_preview() {
    MyApplicationTheme {
        finish_activity()
    }
}
